### PR TITLE
V2 Longhaul scripts

### PR DESCRIPTION
**Pull request by vibhansa-msft**
Adding scripts to run v2 longhaul
- Modify these scripts as per local path
- setup these in cronjob to run at regular internvals

### CODE REVIEW

File: test/longhaul/longhaul.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	sleep 2
+
+	if [ `stat -c %s ./restart2.log` -gt 10485760 ]
+	then 
+		echo "`date` Trimmed " > ./restart2.log
+	fi
+	echo "`date`: Restart : `./blobfuse2 --version`" >> ./restart2.log
+
+	# Send email that blobfuse2 has crashed
+	echo "Blobfuse2 Failure" | mail -s "Blobfuse2 Restart" -A ./restart2.log -a "From: longhaul@blobfuse.com" souravgupta@microsoft.com,vibhansa@microsoft.com
```
Comment (vibhansa-msft) [inline test/longhaul/longhaul.sh:51]:
```suggestion
	echo "Blobfuse2 Failure" | mail -s "Blobfuse2 Restart" -A ./restart2.log -a "From: longhaul@blobfuse.com" 
```

File: test/longhaul/longhaul.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	sleep 2
+
+	if [ `stat -c %s ./restart2.log` -gt 10485760 ]
+	then 
+		echo "`date` Trimmed " > ./restart2.log
+	fi
+	echo "`date`: Restart : `./blobfuse2 --version`" >> ./restart2.log
+
+	# Send email that blobfuse2 has crashed
+	echo "Blobfuse2 Failure" | mail -s "Blobfuse2 Restart" -A ./restart2.log -a "From: longhaul@blobfuse.com"
```
Comment (gapra-msft) [inline test/longhaul/longhaul.sh:51]: who does this send it to?

Comment (vibhansa-msft) [reply test/longhaul/longhaul.sh:51]: :) last argument to this command is the "to" list but did not want to expose our internal mail ids outside so while comittin the script just removed those. To use this script you just need to add your mail id at the end of the command and done.

File: test/longhaul/longhaul.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+		fi
+
+		echo "`whoami` : `date` :: `./blobfuse2 --version` Starting stress test " >> ./longhaul2.log
+
+		mem=$(top -b -n 1 -p `pgrep -x blobfuse2` | tail -1)
+		elap=$( ps -p `pgrep -x blobfuse2` -o etime | tail -1)
+		echo $mem " :: " $elap >> ./longhaul2.log
+	
+		rm -rf /home/vibhansa/blob_mnt2/stress	
+		rm -rf /home/vibhansa/blob_mnt2/myfie*
```
Comment (souravgupta-msft) [inline test/longhaul/longhaul.sh:25]: Typo

**Review by souravgupta-msft — APPROVED** [review]

File: test/longhaul/longhaul.sh (hunk 3 above)
Comment (vibhansa-msft) [reply test/longhaul/longhaul.sh:25]: which typo ?

Comment (tasherif-msft) [reply test/longhaul/longhaul.sh:25]: `myfile` at the end

**Comment by tasherif-msft** [comment]
so this would just be run manually when we wanna do a longhaul ruin?

**Review by tasherif-msft — APPROVED** [review]

**Pull request merged**